businessRules/ideas.js

```diff
@@ -20,33 +27,21 @@ function sortedByDonationSumIdeas() {
 }
 
 function createIdeas(header, description, askedSum, userId) {
-    if (!header) {
-        throw new Error('Error. Please, check your header.');
-    }
+    isHeaderValid(header);
```

Comment:
When a function name starts with `is` you expect that function to return a boolean value. And then you would expect that boolean value to be used in an `if`.

That's not the case here, so `isHeaderValid` name doesn't make the most sense. A better name would be `validateHeader` - this function name communicates that you should not expect it to return a value. Hence calling this function and not doing anything with its return value makes sense.

Reply:
Tryed to rename all function.